hack/run_in_cloud.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Note: Tested only with buildah / podman
+
+set -eux
+
+DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
Seems unnecessarily complicated. You build from `${DIR}/../` later, so you're assuming this is built from the source tree, but you know `hack/` isn't a symlink, so there's no need for the cd/pwd. (It shouldn't matter if anything above the checkout dir is a symlink.)

Reply:
done